main/states/led.py

```diff
@@ -14,8 +14,10 @@
 class LED_COLOR:
 
     # Constants
-    MAX_BRIGHTNESS = 0b11111  # Safeguard: Set to a value appropriate for your setup
-    LED_START = 0b11100000  # Three "1" bits, followed by 5 brightness bits
+    # Safeguard: Set to a value appropriate for your setup
```

Comment:
can revert these comments as these were before? These comments lose the descriptiveness when changed to this way.

Reply:
i have revert the comments